spec/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order'
+
+describe Order do
+  let(:dish_double) { double(:dish, :name => "pizza", :price => 10) }
+  let(:dish_class_double) { double(:dish_class, :new => dish_double) }
+  let(:calc_double) { double(:calc, :total => 20) }
+  let(:calc_class_double) { double(:calc_class, :new => calc_double) }
+
+  let(:text_double) { double(:text, :send => :message_sent) }
+  let(:text_class_double) { double(:text_class, :new => text_double) }
+  let(:menu) { double(:menu, :check => dish_double) }
+  subject { described_class.new(dish_class_double, calc_class_double, text_class_double) }
```

Comment:
Nice job isolating these unit tests from the actual Dish, Calculator and Texter classes